--- ccid-0.9.1/src/ifdhandler.c.orig	2004-06-30 16:06:55.000000000 +0200
+++ ccid-0.9.1/src/ifdhandler.c	2004-08-02 11:50:23.000000000 +0200
@@ -771,8 +771,8 @@
 } /* IFDHTransmitToICC */
 
 
-RESPONSECODE IFDHControl(DWORD Lun, DWORD dwControlCode, LPCVOID TxBuffer,
-	DWORD TxLength, LPVOID RxBuffer, DWORD RxLength, PDWORD pdwBytesReturned)
+RESPONSECODE IFDHControl(DWORD Lun, DWORD dwControlCode, PUCHAR TxBuffer,
+	DWORD TxLength, PUCHAR RxBuffer, DWORD RxLength, PDWORD pdwBytesReturned)
 {
 	/*
 	 * This function performs a data exchange with the reader (not the
